bin/xud

```diff
@@ -154,6 +154,11 @@ const { argv } = require('yargs')
       default: undefined,
       alias: 't',
     },
+    'p2p.torSocks': {
+      describe: 'Port that Tor\'s exposed SOCKS5 proxy is listening on.',
+      type: 'number',
+      default: undefined,
```

Comment:
I don't think it hurts, but I don't think the default is needed for numbers. Undefined is the built in default.